support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -135,7 +135,7 @@ export const tests: Tests = {
 		referrerControlled: false, // ab-test name not needed to be in paramURL
 		seed: 5,
 		targetPage: pageUrlRegexes.contributions.genericCheckoutOnly,
-		excludeCountriesSubjectToContributionsOnlyAmounts: true,
+		excludeContributionsOnlyCountries: true,
 	},
 	adFreeTierThree: {
```

Comment:
Looks like we're unintentionally reenabling `adFreeTierThree` as a global test in this PR

Reply:
Hmmm, not sure how this occured, great spot, thanks